Library/Homebrew/utils.rb

```diff
@@ -404,6 +404,30 @@ def gzip(*paths)
     end
   end
 
+  def reproducible_gzip(*paths)
```

Comment:
Feels a little odd to take multiple `paths` here rather than a single one. Thoughts?

Also, might be nice to add a Sorbet signature here.

Reply:
I'd be fine with this just accepting a single path rather than arbitrarily many. Some formulae do pass in an array of paths already so to transition them to this new function would involve adding an `.each` loop in the formulae. Not a big deal but just wanted to mention that the multiple `paths` form of the existing `gzip` function _is_ currently being used.